schema.sql

```diff
@@ -25,4 +25,7 @@ CREATE TABLE VETS(ID INT GENERATED ALWAYS AS IDENTITY, NAME VARCHAR(60), AGE INT
 
 CREATE TABLE specializations(VETS_ID INT REFERENCES VETS (ID), SPECIES_ID INT REFERENCES SPECIES (ID));
 
-CREATE TABLE visits(VETS_ID INT REFERENCES VETS(ID), ANIMAL_ID INT REFERENCES ANIMALS(ID), VISIT_DATE DATE );
\ No newline at end of file
+CREATE TABLE visits(VETS_ID INT REFERENCES VETS(ID), ANIMAL_ID INT REFERENCES ANIMALS(ID), VISIT_DATE DATE );
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
```

Comment:
## OPTIONAL 
* I strongly suggest that you add indexes for FOREIGN KEYS or on columns that are frequently used in queries as this is one of the basic solutions for the performance problems, I marked this as optional because the previous CR missed this. ( ie CREATE INDEX index_name ON table_name (column1, column2, ...); ). Kindly apply this and see improvement in execution time 🙏 [Read More here](https://www.w3schools.com/sql/sql_create_index.asp)